fs/expose/dokan/__init__.py

```diff
@@ -262,12 +309,13 @@ def wrapper(self,*args):
 
 MIN_FH = 100
 
+
 class FSOperations(object):
     """Object delegating all DOKAN_OPERATIONS pointers to an FS object."""
 
-    def __init__(self, fs, fsname="Dokan FS", volname="Dokan Volume"):
+    def __init__(self, fs, fsname="NTFS", volname="Dokan Volume"):
```

Comment:
What impact does this change have, given that pyfilesystem doesn't support all of NTFS's features? Or was it simply a necessary change because of the newer Dokan?